kafkajs/README.md

```diff
@@ -0,0 +1,80 @@
+# Sample Kafkajs application 
+
+This example provides both a kafkajs producer and a background consumer.
+
+## Getting started
+**Note**: This application requires the use of Node.js v20+ and docker.
+
+ 1. Clone or fork this repository.
```

Comment:
The embedded code blocks break the auto-numbering feature. So we need to number them manually. Auto-numbering can work if the code blocks are indented by at least 4 spaces.